index.html

```diff
@@ -59,212 +59,217 @@
-        );
-    }
-    $.getJSON( "/json/members.json", function( data ) {
-      $.each( data, function( key, val ) {
-        showContributor(val);
-      });
+          '<figure class="uk-overlay uk-overlay-hover contrib-gray">'+ ( contributor.avatar_url ? '<img width="150" src="'+contributor.avatar_url+'">'
+            : '<img width="150" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mO4/R8AArgB255xVG4AAAAASUVORK5CYII=">' )+
+          '<figcaption class="uk-overlay-panel uk-overlay-background uk-overlay-fade uk-flex uk-flex-center uk-flex-middle uk-text-center"><div>'+contributor.login+'<br />'+( contributor.contributions?contributor.contributions+' points':"")+'</div>'+
+          '</figcaption><a class="uk-position-cover" href="'+contributor.html_url+'"></a></figure></div>'
+          );
+      }
+      $.getJSON( "/json/members.json", function( data ) {
+        $.each( data, function( key, val ) {
+          showContributor(val);
+        });
 
-    });
+      });
 
-    function showUser( user ) {
+      function showUser( user ) {
         if (!user) return "";
         return '<div data-uk-tooltip title="'+user.name+'" style="float:left;width:50px;margin:5px 5px 0 0;overflow:hidden">'
         +'<a href="'+user.html_url+'" target="_blank"><img src="'+ user.avatar_url+'" width="100%" /></a></div>';
-    }
-
-    function showFeatureIcon(icon) {
-      var title = "fertiggestelltes Projekt";
-      var color="orange";
-      if (icon == "flash") {
-        color="#f33";
-        title ="derzeit kaputt";
       }
-      if (icon == "cloud-upload") {
-        color="#39f";
-        title ="..work in progress..";
-      }
-      if (icon == 'gears') {
-        color="#88b";
-        title ="In der Planung";
-      }
-      return '<div style="width:1px;height:1px"><div data-uk-tooltip title="'+title+'" style="position:relative;left:5px;top:-25px;color:'+color+';font-size:50px"><i class="uk-icon-'+icon+'"></i></div></div>'
 
-    }
+      function showFeatureIcon(icon) {
+        var title = "fertiggestelltes Projekt";
+        var color="orange";
+        if (icon == "flash") {
+          color="#f33";
+          title ="derzeit kaputt";
+        }
+        if (icon == "cloud-upload") {
+          color="#39f";
+          title ="..work in progress..";
+        }
+        if (icon == 'gears') {
+          color="#88b";
+          title ="In der Planung";
+        }
+        return '<div style="width:1px;height:1px"><div data-uk-tooltip title="'+title+'" style="position:relative;left:5px;top:-25px;color:'+color+';font-size:50px"><i class="uk-icon-'+icon+'"></i></div></div>'
 
-    function showProject( container, p, noimage ) {
-        var percent = p.closed_tasks  / p.total_tasks  * 100;
+      }
+
+      function showProject( container, p, noimage ) {
+        var percent = 0.0;
+        if(p.total_tasks == 0) {
+          percent = 100;
```

Comment:
Hier einfach auf 100 % zu setzen seh ich als ein bisschen problematisch. Nur weil keine Issues/Tasks in einem Repository sind, heisst es ja nicht dass es zu 100% fertig ist.

Reply:
Mh Stimmt. Aber einfach auf 0 zu setzten ist auch doof :/